labgrid/resource/dediprogflasher.py

```diff
@@ -0,0 +1,16 @@
+import attr
+
+from ..factory import target_factory
+from .common import NetworkResource, Resource
+
+
+@target_factory.reg_resource
+@attr.s(eq=False)
+class DediprogFlasher(Resource):
+    """vcc: '3.5V', '2.5V' or '1.8V'"""
+    vcc = attr.ib(validator=attr.validators.instance_of(str))
```

Comment:
```suggestion
    vcc = attr.ib(validator=attr.validators.in_(('3.5V', '2.5V', '1.8V')))
```
See [attrs.validators.in_()](https://www.attrs.org/en/stable/api.html#attrs.validators.in_).